reana-ui/src/components/WorkflowList.js

```diff
@@ -39,7 +38,8 @@ export default class WorkflowList extends Component {
     column: null,
     data: [],
     direction: null,
-    interval: null
+    interval: null,
+    token: this.props.token
```

Comment:
Here you need to wrap the state in the constructor and call super(props) otherwise this.props will return undefined. Generally is a good practice to declare the component state inside a constructor with this.state =  {...} and not as a global.